requestlog/requestlog_test.go

```diff
@@ -75,6 +80,9 @@ func TestHandler(t *testing.T) {
 	if ent.ResponseBodySize != int64(len(responseMsg)) {
 		t.Errorf("ResponseBodySize = %d; want %d", ent.ResponseBodySize, len(responseMsg))
 	}
+	if ent.TraceID == "" {
```

Comment:
I think we have enough information to verify the trace ID matches the one in the span, right? IIUC that would give us more confidence in this test than just testing for non-empty.